tests/test_synced_collection.py

```diff
@@ -154,6 +154,19 @@ def test_update(self, synced_dict, testdata):
         synced_dict.update(d)
         assert len(synced_dict) == 1
         assert synced_dict[key] == d[key]
+        synced_dict.update()
+        assert len(synced_dict) == 1
+        assert synced_dict[key] == d[key]
+        synced_dict.update(update2=testdata)
+        assert len(synced_dict) == 2
+        assert synced_dict['update2'] == testdata
+        synced_dict.update({key: 1}, update=2)
```

Comment:
Can you make a short comment here indicating that `key` is a string `update`? I did a double take reading this test and had to go back to realize why `synced_dict[key] == 2` and not 1 after this test (I didn't immediately realize `key` wasn't quoted and was actually a variable holding the string `'update'`).